Mathlib/Tactic/StacksAttribute.lean

```diff
@@ -0,0 +1,53 @@
+/-
+Copyright (c) 2024 Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa
+-/
+import Lean.Attributes
+
+/-!
+The `stacks` attribute.
+
+This allows tagging of mathlib lemmas with the corresponding
+[Tags](stacks.math.columbia.edu/tags) from the Stacks Project.
```

Comment:
I'm not sure if doc-gen4 is smart enough to link to the right place without https, but probably safest to include it anyways (also below):
```suggestion
[Tags](https://stacks.math.columbia.edu/tags) from the Stacks Project.
```